bok_choy/page_object.py

```diff
@@ -500,3 +520,130 @@ def wait_for_element_invisibility(self, element_selector, description, timeout=6
 
         """
         self.wait_for(lambda: not self.q(css=element_selector).visible, description=description, timeout=timeout)
+
+    def axs_audit_rules(self):
+        """
+        List of rules to check for accessibility errors on the page.
+        See https://github.com/GoogleChrome/accessibility-developer-tools/tree/master/src/audits
+
+        E.g. return ['badAriaAttributeValue']
+        An empty list means to check for all available rules.
+        None means that no audit should be done for this page.
+        """
+        return None
+
+    def axs_scope(self):
+        """
```

Comment:
I agree.  Would this allow us to do something like measure the accessibility of a specific xblock in the platform?